<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1430</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPlainTextEdit" name="plainTextEditInput">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>50</y>
      <width>171</width>
      <height>141</height>
     </rect>
    </property>
    <property name="plainText">
     <string>plainTextEditInput</string>
    </property>
   </widget>
   <widget class="QSpinBox" name="spinBoxCount">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>60</y>
      <width>91</width>
      <height>29</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonGenerate">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>100</y>
      <width>90</width>
      <height>29</height>
     </rect>
    </property>
    <property name="text">
     <string>Generate</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonBuild">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>60</y>
      <width>90</width>
      <height>29</height>
     </rect>
    </property>
    <property name="text">
     <string>Build</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonClear">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>100</y>
      <width>90</width>
      <height>29</height>
     </rect>
    </property>
    <property name="text">
     <string>Clear</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelOutput">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>140</y>
      <width>501</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>Root Outout</string>
    </property>
   </widget>
   <widget class="QPlainTextEdit" name="plainTextEditTree">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>200</y>
      <width>611</width>
      <height>161</height>
     </rect>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
    <property name="plainText">
     <string>Full tree Output</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEditBlockHash">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>60</y>
      <width>171</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>Enter Testnet block hash</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonFetch">
    <property name="geometry">
     <rect>
      <x>600</x>
      <y>60</y>
      <width>90</width>
      <height>29</height>
     </rect>
    </property>
    <property name="text">
     <string>Fetch Block</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>710</x>
      <y>60</y>
      <width>101</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Leaf Index</string>
    </property>
   </widget>
   <widget class="QSpinBox" name="spinBoxVerifyIndex">
    <property name="geometry">
     <rect>
      <x>830</x>
      <y>60</y>
      <width>71</width>
      <height>29</height>
     </rect>
    </property>
    <property name="maximum">
     <number>1000</number>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonVerify">
    <property name="geometry">
     <rect>
      <x>750</x>
      <y>110</y>
      <width>90</width>
      <height>29</height>
     </rect>
    </property>
    <property name="text">
     <string>Verify Leaf</string>
    </property>
   </widget>
   <widget class="QPlainTextEdit" name="plainTextEditProof">
    <property name="geometry">
     <rect>
      <x>920</x>
      <y>60</y>
      <width>331</width>
      <height>85</height>
     </rect>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1430</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
